docs/portuguese/how-to-work-on-guide-articles.md

```diff
@@ -1,17 +1,422 @@
 <table>
     <tr>
         <td> Read these guidelines in </td>
-        <td><a href="/CONTRIBUTING.md"> English </a></td>
-        <td><a href="/docs/chinese/CONTRIBUTING.md"> 中文 </a></td>
-        <td><a href="/docs/russian/CONTRIBUTING.md"> русский </a></td>
-        <td><a href="/docs/arabic/CONTRIBUTING.md"> عربى </a></td>
-        <td><a href="/docs/spanish/CONTRIBUTING.md"> Español </a></td>
-        <td><a href="/docs/portuguese/CONTRIBUTING.md"> Português </a></td>
+        <td><a href="/how-to-work-on-guide-articles.md"> English </a></td>
+        <td><a href="/docs/chinese/how-to-work-on-guide-articles.md"> 中文 </a></td>
+        <td><a href="/docs/russian/how-to-work-on-guide-articles.md"> русский </a></td>
+        <td><a href="/docs/arabic/how-to-work-on-guide-articles.md"> عربى </a></td>
+        <td><a href="/docs/spanish/how-to-work-on-guide-articles.md"> Español </a></td>
+        <td><a href="/docs/portuguese/how-to-work-on-guide-articles.md"> Português </a></td>
```

Comment:
These changes are bad, the links don't work afterwards.

Reply:
The table isn't in the original English version. Should I just delete the table all together or leave the href as CONTRIBUTING.md?

Reply:
Yes, either they should work or be removed altogether. I don't know what the standard practice is in other areas of the guide, but this broken middleground does not fit.